operate/client/package.json

```diff
@@ -7,7 +7,7 @@
     "@bpmn-io/element-template-icon-renderer": "0.5.2",
     "@camunda/camunda-composite-components": "0.7.0",
     "@carbon/elements": "11.46.0",
-    "@carbon/react": "1.58.1",
+    "@carbon/react": "1.57.0",
```

Comment:
This is 1.57.0 on `main` now. So you should be able to rebase. Then there should be no changes needed for package.json and yarn.lock